docs/advanced_options.rst

```diff
@@ -275,3 +275,6 @@ Settings
 
 ``MODERATION_MODERATORS``
     Tuple of moderators' email addresses to which notifications will be sent.
+
+``DEFAULT_FROM_EMAIL``
```

Comment:
This settings is not related to this library so maybe we do not need it here?

Reply:
exactly, this is default Django setting, no need for it here

Reply:
removed it